### ISSUE TITLE
Run a Multiple Tasks concurrently doesn't work if there is only a single package in the monorepo

### ISSUE THREAD

**Issue by xcanchal-uz** [lerna/lerna#3672]
## Current Behavior

Having a monorepo with Lerna 6 (with Nx) and npm workspaces that contains only a single package (named `@scope/package-name`), and trying to execute the following command from the root of the repo, which should execute all the comma-delimited scripts in parallel as stated in the [docs](https://lerna.js.org/docs/features/run-tasks#run-a-multiple-tasks-concurrently), I'm getting the following error:

<img width="783" alt="Screenshot 2023-05-03 at 13 32 43" src="https://user-images.githubusercontent.com/19903051/235905891-c4b93963-91df-4e4c-a47b-20c9ae8a9671.png">

_Note: It seems to interpret it as a single task, because I tried adding a script named `@scope/package-name:format,lint,depcheck,test` in the package's package.json to check if that was happening, and it has been executed indeed_

If I add a second package (named `test`), it works:
 
<img width="728" alt="Screenshot 2023-05-03 at 13 32 34" src="https://user-images.githubusercontent.com/19903051/235908464-d5ca8db0-faac-4a9b-b65b-9c0db9e3cefc.png">

## Expected Behavior

Executing a set of comma-delimited scripts with `lerna run` should execute them even if there is only a single package in the monorepo.

## Steps to Reproduce

1. Create a monorepo with Lerna 6 (with Nx) and npm workspaces that contains only a single package. 

2. Implement a couple of scripts in the package.json of the only existing package as follows:

```json
[...]
"scripts": {
  "hello": "echo 'hello'",
  "bye": "echo 'bye'"
}
[...]
```

3. Execute the following command from the root of the repo:

```bash
lerna run hello,bye
```

4. See the error

### Failure Logs / Configuration

Error screenshots have been added above.

<details><summary>lerna.json</summary><p>

```json
{
	"$schema": "node_modules/lerna/schemas/lerna-schema.json",
	"useWorkspaces": true,
	"version": "independent",
}
```

</p></details>

<details><summary>lerna-debug.log</summary><p>

</p></details>

### Environment

```bash
 Environment info:

  System:
    OS: macOS 13.2.1
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
  Binaries:
    Node: 16.19.1 - ~/.nvm/versions/node/v16.19.1/bin/node
    npm: 8.19.3 - ~/.nvm/versions/node/v16.19.1/bin/npm
  Utilities:
    Git: 2.37.1 - /usr/bin/git
  npmPackages:
    lerna: ^6.5.1 => 6.6.1
```

**Comment by fahslaj** [comment]
Hi @xcanchal-uz , thank you for opening this issue. I believe the problem has been fixed in this PR  (https://github.com/lerna/lerna/pull/3620) and will be released with the next version of Lerna. I'll report back after the version with the fix has been published.

**Comment by xcanchal-uz** [comment]
Hi @fahslaj, many thanks for the quick response. Looking forward to that new version!

**Comment by fahslaj** [comment]
@xcanchal-uz The fix is in the latest Lerna release, 6.6.2: https://github.com/lerna/lerna/releases/tag/6.6.2
